invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/deposit.less

```diff
@@ -156,3 +156,72 @@
   display: inline-block;
   margin-right: 0.5em;
 }
+
+/* Styling for the community selection modal */
+
+#community-selection-modal {
+  margin: 0;
+
+  .community-list-results.content {
+    height: calc(55vh - 10em);
+  }
+
+  .community-list-result-item {
+    .header {
+      white-space: normal;
+      text-overflow: ellipsis;
+      overflow: hidden;
+    }
+  }
+
+  .segment {
+    box-shadow: none;
+  }
+
+  .community-selected-item-container {
+    height: 8rem;
```

Comment:
the height is quite specific, what is the reason of setting those heights? can you should screenshots from "before" custom changes in styling ?

Reply:
sure!.
![Screenshot from 2022-03-07 11-37-42](https://user-images.githubusercontent.com/55200060/157016086-79d73960-abae-4364-b2c3-3835584de1b4.png)